presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestPushPartialAggregationThroughExchange.java

```diff
@@ -197,7 +199,7 @@ public void testPartialAggregationEnabledWhenNotConfident()
                 .overrideStats("values", PlanNodeStatsEstimate.builder()
                         .setOutputRowCount(1000)
                         .addVariableStatistics(variable("b", DOUBLE), new VariableStatsEstimate(0, 100, 0, 8, 800))
-                        .setConfident(false)
+                        .setConfidence(FACT)
```

Comment:
this should be LOW (and if the test passes without that then either the test or the code is broken).

Reply:
why? 1000 values row count is a fact for VALUES clause?

Reply:
because this test is testing that partialaggregation is enabled when stats are not confident. So if the test passes when the stats *are* confident, then either the tests isn't testing what it's supposed to or something is wrong with the code.